frontend/src/components/search/SearchFilterAccordion/SearchFilterAccordion.tsx

```diff
@@ -84,6 +90,8 @@ export function SearchFilterAccordion({
                 mounted={mounted}
                 updateCheckedOption={toggleOptionChecked}
                 toggleSelectAll={toggleSelectAll}
+                isSectionAllSelected={isSectionAllSelected[option.id]}
+                isSectionNoneSelected={isSectionNoneSelected[option.id]}
               />
```

Comment:
Nice.

Reply:
Yea the logic to setup those values is kind of a lot but once you have it it's very easy to use in the components